### DISCUSSION TITLE
Adding keycloak authentication as client credentials grant for ASP.NET Framework Web API

### DISCUSSION THREAD

**Discussion by ambar-ray** [keycloak/keycloak#29092] [Ideas]
I am not getting much  information in the internet regarding adding Keycloak authentication as client credentials grant for ASP.NET Framework Web API using OWIN. Could you please provide some good pointers or guidance.

**Comment by codespearhead** [comment]
I'm afraid this is something that should be asked in the respective framework's community, since in terms of system integration, Keycloak behaves just like any other OAuth2-compliant server.

So the question is how to communicate with OAuth2 servers using the OIDC-compliant client that your framework offers.
